homeassistant/components/trest_solar/sensor.py

```diff
@@ -0,0 +1,133 @@
+"""Entity that represents the trest solar controller and its data values."""
+
+from datetime import datetime
+
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+)
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.device_registry import DeviceEntryType, DeviceInfo
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from . import TrestConfigEntry
+from .const import DOMAIN
+from .coordinator import TrestDataCoordinator
+
+SENSOR_TYPES: tuple[SensorEntityDescription, ...] = (
+    SensorEntityDescription(
+        key="battery_discharge",
+        name="Battery Discharge",
```

Comment:
Please use translation keys

Reply:
What is translation keys? How do I implement them?